README.md

```diff
@@ -87,6 +87,12 @@ The Ginkgo CUDA module has the following __additional__ requirements:
 The Ginkgo OMP module has the following __additional__ requirements:
 *  _MinGW_ or _Cygwin_
 
+Need to add the following manually:
+* Test Ginkgo with shared Ginkgo library:
+  Add `PROJECT_BINARY_DIR/GINKGO_WINDOWS_SHARED_LIBRARY_RELPATH` into the environment `PATH`. `GINKGO_WINDOWS_SHARED_LIBRARY_RELPATH` is `windows_shared_library` by default. More Details are availble in the [Installation page](./INSTALL.md).
+* Build Ginkgo with Debug mode:
+  Add `\bigobj` in _Microsoft Visual Studio_ or `-Wa,-mbig-obj -O1` in _MinGW_, _Cygwin_ into complier flag.
```

Comment:
wording suggestion/typos:
```suggestion
  Add `\bigobj` in _Microsoft Visual Studio_ or `-Wa,-mbig-obj -O1` in _MinGW_, _Cygwin_ to the compiler flag.
```
Is `-O1` still necessary, or did we only do that due to space constraints in Github Actions? Also, could you mention how to set the compiler flags?

Reply:
Yes, `-O1` is necessary. The error is from ld (error: export ordinal too large (some number >= 65536)) 
We can not compile Debug + Static on Github Action due to space issue